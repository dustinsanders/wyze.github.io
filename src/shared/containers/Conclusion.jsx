// @flow

import type { Style, VNode } from '../../types'
import { css } from 'glamor'
import { thin } from '../styles'
import Box from '../components/Box'
import H2 from '../components/H2'
import Link from '../components/Link'
import Section from '../components/Section'
import preact from 'preact'

const conclusion: Style = css({
  width: '100%',
})

const Conclusion = (): VNode => (
  <Box>
    <Section styles={conclusion}>
      <H2 styles={thin}>
        Download{' '}
        <Link to="/resume.pdf">résumé</Link>.
        {' '}View{' '}
        <Link to="//github.com/wyze/wyze.github.io">source</Link>.
      </H2>
    </Section>
  </Box>
)

export default Conclusion
