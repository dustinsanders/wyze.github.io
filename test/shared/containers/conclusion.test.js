// @flow

import Conclusion from '../../../src/shared/containers/Conclusion'
import snapshot from '../../helpers/snapshot'

describe('<Conclusion />', () => {
  it('renders', snapshot.bind(null, Conclusion))
})
