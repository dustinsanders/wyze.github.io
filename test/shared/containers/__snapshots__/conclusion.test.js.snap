// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Conclusion /> renders 1`] = `
"<Box>
  <Section styles=\\"css-63oe3q\\">
    <H2 styles=\\"css-1wdk04o\\">
      Download 
      <Link to=\\"/resume.pdf\\">résumé</Link>
      . View 
      <Link to=\\"//github.com/wyze/wyze.github.io\\">source</Link>
      .
    </H2>
  </Section>
</Box>"
`;
